main/aspell-fr/APKBUILD

```diff
@@ -1,28 +1,28 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=aspell-fr
-pkgver=0.50_p3
-_ver=${pkgver/_p/-}
+pkgver=0.50
 pkgrel=1
+_lang=${pkgname##*-}
+_name=${pkgname%%-*}-$_lang-$pkgver-3
```

Comment:
is the last `-3` a fixed number or does it have a meaning (eg like patch level?) if its patch level then we should use `pkgver=0.50_p3`